### PR TITLE
Fixes #26

### LINKED ISSUE #26: Add Windows support
_From @alexsaveliev on May 6, 2016 9:58_

I see no reason of not having Windows support for srclib-javascript. 

- Please make sure that everywhere where file path are emitted, Unix-style separators (slash) are used to produce test results consistent with the expected data
- Please add `srclib-javascript.bat` file to run nodejs with the proper arguments 

_Copied from original issue: alexsaveliev/srclib-javascript#3_

### PR DESCRIPTION
**Pull request by alexsaveliev**
- fixes #26 by adding .bat runner for Windows and converting file paths to Unix style

### CODE REVIEW

File: lib/define-path-info.js [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
     return fileName + "_" + data.start + "_" + data.end;
 }
 
 function getFilePathName(fileName) {
     var index = fileName.indexOf("node_modules");
     if (index > -1) {
-        var filePath = path.normalize(fileName);
+        var filePath = util.normalizePath(fileName);
         var pathRes = filePath.split("/");
         //console.error("File path name = ", pathRes.slice(2).join("/"));
```
Comment (beyang) [inline lib/define-path-info.js]: remove

**Comment by beyang** [comment]
1 comment, otherwise LGTM

File: lib/define-path-info.js (hunk 1 above)
Comment (alexsaveliev) [inline lib/define-path-info.js]: Sorry, but I don't understand what should be removed

Comment (beyang) [inline lib/define-path-info.js]: remove the comment line `//console.error`

**Pull request merged**